scripts/init_tiff_to_zarr_globus_flow.py

```diff
@@ -31,9 +31,6 @@ def conversion_wrapper(rundir, h5_file_name, folder_path):
     """
     import os
     import subprocess
-    import time
```

Comment:
Was the issue that references time.run() at the import or the use of `time.time()`

That's super duper odd. I would be interested to see `print(time.time)` if it's no occuring during the import.

Reply:
I couldn't recreate the issue locally but only when the Globus Flow called the function. There error listed specifically the line `start = time.time()`. What's odd to me is A) it was working before and B) I am calling time.time() and nowhere do I call time.run().